qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
> The problem with the downloading the file at provisioning time is this operation is done once unless we destroy the machine OR manually run the vagrant provision command.

nope, the situation here is the same as downloading the thing inside the rspec tests. Download should happen 1, then is responsibility of the test framework to keep the right state inside the tests.

> Are we at risk to run the tests not on the latest released version in local dev machine?

nope, as I said, risks are the same independently where do you download the artifact. we should be able to create a URL that lets us point to latest released.

>  but locally as human we won't be doing that since Its really time consuming.

this is specially my point here, I see test framework being sure that the state is the right one, and then we can reuse the download artifacts any time we run the test with the same machine.

> I agree having the spec download the file could be slower, but nothing prevent us from persisting it. 

this is what we get by doing this at provision time.

> before :all 

problem with this is that we download any time the rspec examples are executed, I want us to avoid that by doing this in  provision time.